events-api/src/modules/payment/payment.service.ts

```diff
@@ -0,0 +1,41 @@
+import { HttpService } from "@nestjs/axios";
+import { BadRequestException, Injectable } from "@nestjs/common";
+import { AxiosError } from "axios";
+import { lastValueFrom } from "rxjs";
+
+
+@Injectable()
+export class PaymentService {
+  constructor(private readonly httpService: HttpService){}
+
+  async payingProcess(paymentData: any) {
+    const paymentApi = process.env.PAYMENT_API_URL
+    if (!paymentApi) {
+      throw new BadRequestException('Cannot find payment Api')
```

Comment:
Why you are throwing 400 error? Can client fix this error? I think that it is a server (50x) related error.

Reply:
Fixed